SpatialGDK/Source/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -92,6 +102,32 @@ bool USpatialGameInstance::StartGameInstance_SpatialGDKClient(FString& Error)
 	return bOk;
 }
 
+FURL USpatialGameInstance::GetInitialGameURL() const
+{
+	// Load the default URL from config files.
+	FURL DefaultURL;
+	DefaultURL.LoadURLConfig(TEXT("DefaultPlayer"), GGameIni);
+
+	const TCHAR* Cmd = FCommandLine::Get();
```

Comment:
There is no need to store the Cmd, we could just pass it directly to `FParse::Token`

Reply:
Actually not true, `FParse::Token` takes in a reference to a `TCHAR*`, and actually modifies the pointer. My guess is to advance it past the token you read.